client_tools.py

```diff
@@ -105,6 +111,13 @@ def main(args):
         if args.collection == "Botany_PIC":
             md5_insert = args.md5
             PicturaeUndoBatch(MD5=md5_insert)
```

Comment:
added update option for client tools, on the command line, with update specific "force_update" argument that allows for the database checks to be overriden.